.github/workflows/generate-changelog.yml

```diff
@@ -13,11 +13,48 @@ jobs:
     with:
       script_call: 'make tools update-changelog-unreleased-section'
       file_to_commit: 'CHANGELOG.md'
-      commit_message: ${{ github.event_name == 'workflow_dispatch' && 'Update CHANGELOG.md (Manual Trigger)' || format('{0}{1}', 'Update CHANGELOG.md for \#', github.event.pull_request.number) }}
+      commit_message: ${{ github.event_name == 'workflow_dispatch' && format('{0}{1}', 'chore:',' Updates CHANGELOG.md (Manual Trigger)') || format('{0}{1}{2}{3}', 'chore:', ' Updates CHANGELOG.md for ', '#', github.event.pull_request.number) }}
     secrets:
       apix_bot_pat: ${{ secrets.APIX_BOT_PAT }}
       remote: https://svc-apix-bot:${{ secrets.APIX_BOT_PAT }}@github.com/${{ github.repository }}  
       gpg_private_key: ${{ secrets.APIX_BOT_GPG_PRIVATE_KEY }}
       passphrase: ${{ secrets.APIX_BOT_PASSPHRASE }}     
 
-  
\ No newline at end of file
+  slack-notification:
+    needs: [generate-and-update-changelog]
+    if: ${{ !cancelled() && needs.generate-and-update-changelog.result == 'failure' }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Send Slack message
+        id: slack
+        uses: slackapi/slack-github-action@6c661ce58804a1a20f6dc5fbee7f0381b469e001
+        with:
+          payload: |
+            {
+              "text": "Automatic Changelog update failed",
+              "blocks": [
+                {
+                  "type": "section",
+                  "text": {
+                    "type": "mrkdwn",
+                    "text": "*Automatic Changelog update failed* ${{ secrets.SLACK_ONCALL_TAG }}"
+                  }
+                },
+                {
+                  "type": "actions",
```

Comment:
knit: consider use a link instead of an action, e.g. slack to docs

Reply:
Done!